test/blackbox-tests/test-cases/describe.t

```diff
@@ -215,6 +215,55 @@ Setup
   > EOF
   $ touch subdir/subfolder/subfolder_lib.ml
 
+  $ mkdir ppx_test
```

Comment:
Would it be possible to move this into a separate test file? `describe.t` is becoming quite busy?

A test file like `describe-pp.t` would be much preferable IMO

Reply:
Agreed. I created a file `describe-workspace-pp.t`, so that it is not confused with the already existing `describe-pp.t` tests.